docs/book/provider_implementations/create_actuators.md

```diff
@@ -35,6 +35,9 @@ import (
         client "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
 )
 
+//Add RBAC rules to access cluster-api resources
```

Comment:
Missing space between `//` and first word.

Reply:
Fixed